iotile_analytics_core/iotile_analytics/core/group.py

```diff
@@ -203,13 +203,15 @@ def fetch_stream(self, slug_or_name):
                 can be a partial match to a full stream slug or name so long
                 as it uniquely matches.  This is passed to find_stream so anything
                 that find_stream accepts will be accepted here.
+            allow_empty (bool): Allow fetching an empty stream.  Otherwise an ArgumentError
```

Comment:
Checking the wording here. When will an `ArgumentError` be raised? When the target stream is empty, while `allow_empty` is `True`?

Reply:
Updated the wording:

```
allow_empty (bool): Allow fetching an empty stream.  If allow_empty is False or
       not passed, an ArgumentError will be raised if the target stream is empty.
```